src/test/java/org/cactoos/http/HtTimedWireTest.java

```diff
@@ -60,20 +58,19 @@ public void worksFine() throws Exception {
         );
     }
 
-    @Ignore("see BlockingSocketServer todo for #87")
     // @checkstyle MagicNumberCheck (1 line)
     @Test(expected = TimeoutException.class, timeout = 1000)
+    @SuppressWarnings("PMD.AvoidUsingHardCodedIP")
     public void failsAfterTimeout() throws Exception {
-        // @checkstyle MagicNumberCheck (1 line)
+        // @checkstyle MagicNumberCheck (2 lines)
         final long timeout = 100;
-        try (BlockingSocketServer server = new BlockingSocketServer()) {
-            new HtTimedWire(
-                new HtWire(
-                    server.address().getHostAddress(),
-                    server.port()
-                ),
-                timeout
-            ).send(new InputOf("unused"));
-        }
+        final int port = 80;
+        new HtTimedWire(
+            new HtWire(
+                "192.168.0.0",
+                port
```

Comment:
@iakunin I think the port can stay inlined

Reply:
@victornoel no problem. Inlined.